content/en/docs/tasks/security/ca-config/plugin-ca-cert/index.md

```diff
@@ -7,16 +7,13 @@ aliases:
     - /docs/tasks/security/plugin-ca-cert/
 ---
 
+In Istio 1.5, Citadel is linked with Pilot.
```

Comment:
can this be reworded?   Should it be something like Istio CA from citadel is merged into istiod?

Reply:
+1 - I don't think users whould see 'Citadel' or 'Pilot' in any new docs.

Reply:
Why is this needed at all? It's an implementation detail. How about just remove this line and change the following line to:
```
This task shows how operators can configure the Istio certificate authority with an existing root certificate, signing certificate and key.
```
You also should change all occurrences of `Citadel` to `Istio's CA` in the rest of the doc.

Reply:
Done.